bot/exts/evergreen/wikipedia.py

```diff
@@ -0,0 +1,109 @@
+import asyncio
+import datetime
+import logging
+from typing import List, Optional
```

Comment:
`Optional` is not being used anymore and `flake8` errors out when `pre-commit` hooks are run.